db/migrate/20150420223003_create_admins.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAdmins < ActiveRecord::Migration
+  def change
+    create_table :admins do |t|
+      t.string :username
+      t.string :password
```

Comment:
So I think we should has_secure_password in the Admin model so we don't store the password directly in the database. I think only password_digest should be in there.